database/README.md

```diff
@@ -13,8 +13,8 @@ one entity can have the database open at a time (for most database backends),
 and that entity will be dcrd.
 
 When a client wants programmatic access to the data provided by dcrd, they'll
-likely want to use the [dcrrpcclient](https://github.com/decred/dcrrpcclient)
-package which makes use of the [JSON-RPC API](https://github.com/decred/dcrd/tree/master/docs/json_rpc_api.mediawiki).
+likely want to use the [rpcclient](https://github.com/decred/rpcclient) package
```

Comment:
The updated link doesn't look valid to me.

Reply:
Indeed.  Updated.  Not sure how I managed to transpose that incorrectly.